src/pretalx/submission/models/question.py

```diff
@@ -36,6 +46,12 @@ class Question(LogMixin, models.Model):
         choices=QuestionVariant.get_choices(),
         default=QuestionVariant.STRING,
     )
+    target = models.CharField(
+        max_length=QuestionTarget.get_max_length(),
+        choices=QuestionTarget.get_choices(),
+        default=QuestionTarget.SUBMISSION,
+        verbose_name=_('question type'),
```

Comment:
is it a "type" or is it a "target"?

Reply:
It's a "target" internally, among other things because "type" is a function in python. It is a "type" in the user interface since that is much more understandable.

Reply:
welp, okay :P